### PR TITLE
ci: set semantic type as CI and include digests as CI operations

### PR DESCRIPTION
**Pull request by cmeister2**
Hopefully removing the last of the "chores" from Renovate.

### CODE REVIEW

File: renovate.json
Comment (vszakats) [inline renovate.json:86]: Minor nit: A newline would nice here.

Comment (bagder) [reply renovate.json:86]: agreed

**Review by bagder — APPROVED** [review]
Apart from the little nit, this looks good

File: renovate.json
Comment (cmeister2) [reply renovate.json:86]: Will fix - auto formatter got a bit enthusiastic I think

**Comment by cmeister2** [comment]
Fix pushed, will merge after checks complete.

**Comment by jay** [comment]
usually in a pr you would squash your incremental work together into one commit, unless they are truly separate, and then rebase before landing. you can judge whether that is the case here. your commit will need an explanation and a reference most of the time. references have a separate blank line and then `Closing #xxxx` and/or `Ref: url` etc. please review [push-access-guidelines](https://github.com/curl/curl/wiki/push-access-guidelines) for more detail. it may help to go through a bit of the repo commit history with `gitk --all` + down button to quickly get an idea of what is acceptable

**Comment by cmeister2** [comment]
> usually in a pr you would squash your incremental work together into one commit, unless they are truly separate, and then rebase before landing. you can judge whether that is the case here. your commit will need an explanation and a reference most of the time. references have a separate blank line and then `Closing #xxxx` and/or `Ref: url` etc. please review [push-access-guidelines](https://github.com/curl/curl/wiki/push-access-guidelines) for more detail. it may help to go through a bit of the repo commit history with `gitk --all` + down button to quickly get an idea of what is acceptable

I've rebased the nit comment. I don't think there are any URLs to reference here. If I'm doing pushes wrong I'm happy to let someone with a bit more experience handle the merge here.

**Comment by cmeister2** [comment]
> > If I'm doing pushes wrong
> 
> The `Closes #[number]` line was missing in some previous commits.

Ah ok - I didn't have a corresponding issue created for this work.

**Comment by bagder** [comment]
It should say `Closes`  for the PR and `Fixes` for the issue, if there is one.

**Pull request closed without merging**
